### ISSUE TITLE
Broken unit tests with new consensus

### ISSUE THREAD

**Issue by platfowner** [ainblockchain/ain-blockchain#44]
```
./node_modules/mocha/bin/mocha --timeout 80000 -R  spec ./test/block-pool.test.js

...
    1) Returns a finalizable chain
...
    2) Correctly returns the longest notarized chain


  2 passing (261ms)
  2 failing

  1) BlockPool
       Returns a finalizable chain:
     AssertionError: expected undefined to equal true
      at Context.it (test/block-pool.test.js:126:65)

  2) BlockPool
       Correctly returns the longest notarized chain:

      AssertionError: expected [ Array(1) ] to deeply equal [ Array(1) ]
      + expected - actual

               "0x03AAb7b6f16A92A1dfe018Fe34ee420eb098B98A": 250
               "0x04A456C92A880cd59D7145C457475515a6f6E0f2": 250
             }
           }
      +    {
      +      "epoch": 1
      +      "hash": "0x061063b295628fc0e8bcffee5455c50a8287ccc38909f847f211dc6176bf637d"
      +      "last_hash": "0x559e24b3e9410e522e2804067dc7104fccbdaf84397fb5024645759f2b9df977"
      +      "last_votes": []
      +      "last_votes_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "number": 1
      +      "proposer": "0x00ADEc28B6a845a085e03591bE7550dd68673C1C"
      +      "size": 0
      +      "timestamp": 1596701907175
      +      "transactions": []
      +      "transactions_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "validators": {
      +        "0x00ADEc28B6a845a085e03591bE7550dd68673C1C": 250
      +      }
      +    }
      +    {
      +      "epoch": 3
      +      "hash": "0x30f1bd1a246ea74e603dce7741c722434884f4c2b853bd4d37299521b8dc65b2"
      +      "last_hash": "0x061063b295628fc0e8bcffee5455c50a8287ccc38909f847f211dc6176bf637d"
      +      "last_votes": []
      +      "last_votes_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "number": 2
      +      "proposer": "0x00ADEc28B6a845a085e03591bE7550dd68673C1C"
      +      "size": 0
      +      "timestamp": 1596701907176
      +      "transactions": []
      +      "transactions_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "validators": {
      +        "0x00ADEc28B6a845a085e03591bE7550dd68673C1C": 250
      +      }
      +    }
      +    {
      +      "epoch": 5
      +      "hash": "0x4ee0a60f12a197e9ecee35e3d9f4e1c82900c403b177b85a7ca797bf7f47396d"
      +      "last_hash": "0x30f1bd1a246ea74e603dce7741c722434884f4c2b853bd4d37299521b8dc65b2"
      +      "last_votes": []
      +      "last_votes_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "number": 3
      +      "proposer": "0x00ADEc28B6a845a085e03591bE7550dd68673C1C"
      +      "size": 0
      +      "timestamp": 1596701907178
      +      "transactions": []
      +      "transactions_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "validators": {
      +        "0x00ADEc28B6a845a085e03591bE7550dd68673C1C": 250
      +      }
      +    }
      +    {
      +      "epoch": 7
      +      "hash": "0xc3b2e67f021322f9301810d9347d170508d30a1c58926b876786f89b1184a347"
      +      "last_hash": "0x4ee0a60f12a197e9ecee35e3d9f4e1c82900c403b177b85a7ca797bf7f47396d"
      +      "last_votes": []
      +      "last_votes_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "number": 4
      +      "proposer": "0x00ADEc28B6a845a085e03591bE7550dd68673C1C"
      +      "size": 0
      +      "timestamp": 1596701907179
      +      "transactions": []
      +      "transactions_hash": "0x853fb99c831d4952ff90b897bd7d7c5c2f3747e8eda8ad13e7359b731eadc299"
      +      "validators": {
      +        "0x00ADEc28B6a845a085e03591bE7550dd68673C1C": 250
      +      }
      +    }
         ]
       ]
      
      at Context.it (test/block-pool.test.js:201:12)
```

**Comment by minsulee2** [comment]
it's also been close init? :)

**Comment by liayoo** [comment]
Yup, fixed in #47. Will close the issue.
